.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+jobs:
+  build:
+    machine:
+      image: circleci/classic:201808-01  # pins image to specific version using YYYY-MM format
+    steps:
+      - checkout
+      - run:
+          name: Run AMQP integration specs
+          command: docker-compose up --abort-on-container-exit protein-client-amqp
+      - run:
+          name: Run HTTP integration specs
+          command: docker-compose up --abort-on-container-exit protein-client-http
```

Comment:
Multiple files in this PR have no trailing newline resulting in warning marks from git & github. This is considered a malformed file by git and it pollutes diffs later on if left in such shape. You can definitely pull this off automatically on save with proper editor config / plugin. In Sublime it's `ensure_newline_at_eof_on_save`.

Reply:
I think I have misconfigured new IDE. I'll fix this.